platform-sdk/platform-apps/tests/StressTestingTool/settings.gradle.kts

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright (C) 2020-2023 Hedera Hashgraph, LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-pluginManagement { includeBuild("../../../../build-logic") }
-
-includeBuild("../../../../hedera-dependency-versions")
-
-includeBuild("../../../../platform-sdk")
```

Comment:
This file is unused code - leftover that we missed during the Gradle migration steps